src/test/java/org/candlepin/subscriptions/db/HostRepositoryTest.java

```diff
@@ -539,6 +541,50 @@ void testShouldFilterCores() {
         assertEquals(1, physical.getCores());
     }
 
+    @Transactional
+    @ParameterizedTest
+    @CsvSource({
```

Comment:
At some point I'd like to look into bumping our JUnit version.  With newer minor versions, there are more options available for parameterized tests - like more defined/obvious ways of passing around null sources.

There's also an option (and available with our current version I think) to use this `@CsvSource` with an external resource file instead.  Anyone have thoughts or preference on that?  Personally, I think it's easier to read with the data in-line like this to prevent the need to open and switch between another file.

Reply:
I dig in-line here. I'd say a rule of thumb would be appropriate when reviewing. I would go with more than 10 lines of CSV should be in a separate file.